src/components/views/settings/tabs/user/EncryptionUserSettingsTab.tsx

```diff
@@ -54,71 +61,115 @@ interface EncryptionUserSettingsTabProps {
     initialState?: State;
 }
 
+const useKeyBackupIsEnabled = (): boolean | undefined => {
+    const [isEnabled, setIsEnabled] = useState<boolean | undefined>(undefined);
+    const [loading, setLoading] = useState(true);
```

Comment:
Is `loading` redondant when `isEnabled = undefined`?

It looks like we can remove `loading` and just returns `isEnabled`

Reply:
True, it maybe felt a bit more explicit but it is unnecessary